src/routes/loginOAuth.js

```diff
@@ -40,4 +40,43 @@ server.post("/auth/login", async (req, res) => {
   }
 });
 
+server.post("/auth/changePassword", async (req, res) => {
+  const { username, oldPassword, newPassword } = req.body;
+  let encryptedPassword
+  await encrypt.encryptPassword(newPassword).then(encryptedPass => {
+    encryptedPassword = encryptedPass;
+  })
+
+  const user = await Users.findOne({ username });
+  if (!user) {
+    res.status(401).json({
+      message: "Username not found",
+      error: "401: User not found",
+    });
+  } else {
+
+    if (user.password != undefined) {
+      let validPassword;
+      await encrypt.comparePassword(oldPassword, user.password).then((res) => {
+        validPassword = res
+      });
+
+      if (!validPassword) {
+        res.status(402).json({
```

Comment:
Same principle here as above, we can get rid of the else statement by returning res.status if the password is invalid so the code down below will not execute
```suggestion
        return res.status(402).json({
```